aten/src/ATen/native/tags.yaml

```diff
@@ -82,6 +82,14 @@
           Pointwise operators are operators where each element of the output is computed only by accessing
           the corresponding element of all the broadcasted inputs. The output shape will be the broadcasted
           shape of the inputs.
+- tag: reduction
+  desc: |
+          Reduction operations aggregate tensor elements along one or more dimensions,
+          producing an output with fewer dimensions.
+- tag: view
+  desc: |
+          This tag indicates that the operator creates a pure view/alias Tensor and has an explicit
+          derivative formula in derivates.yaml.
```

Comment:
hmm view tag is probably less critical / a bit redundant, given that `OpOverload` objects already tell you whether they are a view based off of their schema (you can run `op_overload.is_view`, see [link](https://github.com/pytorch/pytorch/blob/main/torch/_ops.py#L776))

I guess the counterargument is that tags are more a general-purpose of grouping ops into categories, and it's easier for a pass writer to use tags for everything rather than tags for some metadata and schema info for other metadata. So I'm not too opinionated, although we should at least ensure that our OpInfo tests assert that the tag always matches the `op_overload.is_view` "source of truth". cc @eellison

Reply:
I think there should be one source of truth for views. That source of truth should be the schema, not tags. People are not going to add the tags to custom operators.